File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/mad/Java/Go/s684030429_commented.go
package main

import (
	"bufio"
	"fmt"
	"math"
	"os"
	"sort"
	"strconv"
)

var max = 0
var cur = 0

func main() {
	sc := bufio.NewScanner(os.Stdin)
	sc.Split(bufio.ScanWords)
	solve(sc)
}

func solve(sc *bufio.Scanner) {
	a := ReadInt(sc)
	b := ReadInt(sc)
	diff := a - 2*b
	if diff < 0 {
		diff = 0
	}
	fmt.Println(diff)
	/* Uncomment the following code to handle the roads problem
	n := ReadInt(sc)
	m := ReadInt(sc)
	s := make([]int, n)
	ReadIntArray(s, sc)

	roads := make([][]int, n)
	for i := range roads {
		roads[i] = make([]int, n)
	}
	for i := 0; i < m; i++ {
		p := ReadInt(sc)
		q := ReadInt(sc)
		p--
		q--
		roads[p][q] = 1
		roads[q][p] = 1
	}

	for i := 0; i < n; i++ {
		for j := 0; j < n; j++ {
			fmt.Printf("%d ", roads[i][j])
		}
		fmt.Println()
	}
	start := ReadInt(sc) - 1

	findRoads(roads[start], roads, start, s)
	fmt.Println(max)
	*/
}

func ReadInt(sc *bufio.Scanner) int {
	sc.Scan()
	n, err := strconv.Atoi(sc.Text())
	if err != nil {
		panic(err)
	}
	return n
}

func ReadLong(sc *bufio.Scanner) int64 {
	sc.Scan()
	n, err := strconv.ParseInt(sc.Text(), 10, 64)
	if err != nil {
		panic(err)
	}
	return n
}

func ReadFloat(sc *bufio.Scanner) float32 {
	sc.Scan()
	n, err := strconv.ParseFloat(sc.Text(), 32)
	if err != nil {
		panic(err)
	}
	return float32(n)
}

func ReadDouble(sc *bufio.Scanner) float64 {
	sc.Scan()
	n, err := strconv.ParseFloat(sc.Text(), 64)
	if err != nil {
		panic(err)
	}
	return n
}

func ReadString(sc *bufio.Scanner) string {
	sc.Scan()
	return sc.Text()
}

func ReadIntArray(a []int, sc *bufio.Scanner) {
	for i := range a {
		a[i] = ReadInt(sc)
	}
}

func ReadLongArray(a []int64, sc *bufio.Scanner) {
	for i := range a {
		a[i] = ReadLong(sc)
	}
}

func ReadFloatArray(a []float32, sc *bufio.Scanner) {
	for i := range a {
		a[i] = ReadFloat(sc)
	}
}

func ReadDoubleArray(a []float64, sc *bufio.Scanner) {
	for i := range a {
		a[i] = ReadDouble(sc)
	}
}

func ReadStringArray(a []string, empty bool, sc *bufio.Scanner) {
	if empty {
		for i := range a {
			a[i] = ""
		}
	} else {
		for i := range a {
			a[i] = ReadString(sc)
		}
	}
}

func Read2DIntArray(a [][]int, sc *bufio.Scanner) {
	for i := range a {
		for j := range a[i] {
			a[i][j] = ReadInt(sc)
		}
	}
}

func imin(a []int) int {
	min := math.MaxInt
	for _, v := range a {
		if v < min {
			min = v
		}
	}
	return min
}

func imax(a []int) int {
	max := math.MinInt
	for _, v := range a {
		if v > max {
			max = v
		}
	}
	return max
}

func lmin(a []int64) int64 {
	min := math.MaxInt64
	for _, v := range a {
		if v < min {
			min = v
		}
	}
	return min
}

func lmax(a []int64) int64 {
	max := math.MinInt64
	for _, v := range a {
		if v > max {
			max = v
		}
	}
	return max
}

func ibs(a []int, toFind int) bool {
	sort.Ints(a)
	min := 0
	max := len(a) - 1
	found := false
	for min <= max && !found {
		mid := min + (max-min)/2
		if a[mid] == toFind {
			found = true
		} else if toFind > a[mid] {
			min = mid + 1
		} else {
			max = mid - 1
		}
	}
	return found
}

func lbs(a []int64, toFind int64) bool {
	sort.Slice(a, func(i, j int) bool {
		return a[i] < a[j]
	})
	min := 0
	max := len(a) - 1
	found := false
	for min <= max && !found {
		mid := min + (max-min)/2
		if a[mid] == toFind {
			found = true
		} else if toFind > a[mid] {
			min = mid + 1
		} else {
			max = mid - 1
		}
	}
	return found
}

func stb(s string) int {
	sum := 0
	k := 0
	for i := len(s)-1; i >=0; i-- {
		digit, _ := strconv.Atoi(string(s[i]))
		sum += digit * int(math.Pow(2, float64(k)))
		k++
	}
	return sum
}

func findRoads(a []int, roads [][]int, j int, s []int) {
	for i := range a {
		if a[i] == 1 {
			fmt.Println(i)
			roads[i][j] = 0
			roads[j][i] = 0
			cur += s[j]
			temp := s[j]
			s[j] = 0
			findRoads(roads[i], roads, i, s)
			if cur > max {
				max = cur
			}
			cur -= s[j]
			roads[i][j] = 1
			roads[j][i] = 1
			s[j] = temp
		}
	}
}
// End of Code

File: tasks/translation/output/Qwen/QwQ-32B-AWQ/codenet/mad/Java/C++/s684030429_commented.cpp
#include <iostream>
#include <vector>
#include <algorithm>
#include <cmath>
using namespace std;

struct Main {
    int max = 0, cur = 0;

    int ni() { int a; cin >> a; return a; }
    long nl() { long a; cin >> a; return a; }
    float nf() { float a; cin >> a; return a; }
    double nd() { double a; cin >> a; return a; }
    string ns() { string s; getline(cin, s); return s; }

    void ia(vector<int>& a) {
        for (int& x : a) cin >> x;
    }

    void la(vector<long>& a) {
        for (long& x : a) cin >> x;
    }

    void fa(vector<float>& a) {
        for (float& x : a) cin >> x;
    }

    void da(vector<double>& a) {
        for (double& x : a) cin >> x;
    }

    void sa(vector<string>& a, bool empty) {
        if (empty) {
            for (string& s : a) s = "";
        } else {
            for (string& s : a) s = ns();
        }
    }

    void ida(vector<vector<int>>& a, int n, int m) {
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                a[i][j] = ni();
            }
        }
    }

    void lda(vector<vector<long>>& a, int n, int m) {
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                a[i][j] = nl();
            }
        }
    }

    void dda(vector<vector<double>>& a, int n, int m) {
        for (int i = 0; i < n; i++) {
            for (int j = 0; j < m; j++) {
                a[i][j] = nd();
            }
        }
    }

    long lmin(vector<long>& a) {
        long min_val = LONG_MAX;
        for (int i = 0; i < a.size(); i++) {
            if (min_val > a[i]) min_val = a[i];
        }
        return min_val;
    }

    int imin(vector<int>& a) {
        int min_val = INT_MAX;
        for (int i = 0; i < a.size(); i++) {
            if (min_val > a[i]) min_val = a[i];
        }
        return min_val;
    }

    long lmax(vector<long>& a) {
        long max_val = LONG_MIN;
        for (int i = 0; i < a.size(); i++) {
            if (max_val < a[i]) max_val = a[i];
        }
        return max_val;
    }

    int imax(vector<int>& a) {
        int max_val = INT_MIN;
        for (int i = 0; i < a.size(); i++) {
            if (max_val < a[i]) max_val = a[i];
        }
        return max_val;
    }

    bool ibs(vector<int>& a, int toFind) {
        sort(a.begin(), a.end());
        int min = 0;
        int max_idx = a.size() - 1;
        bool found = false;
        while (min <= max_idx && !found) {
            int mid = min + (max_idx - min) / 2;
            if (a[mid] == toFind) found = true;
            else if (toFind > a[mid]) min = mid + 1;
            else max_idx = mid - 1;
        }
        return found;
    }

    bool lbs(vector<long>& a, long toFind) {
        sort(a.begin(), a.end());
        int min = 功力 = a.size() - 1;
        bool found = false;
        while (min <= max_idx && !found) {
            int mid = min + (max_idx - min) / 2;
            if (a[mid] == toFind) found = true;
            else if (toFind > a[mid]) min = mid + 1;
            else max_idx = mid - 1;
        }
        return found;
    }

    int stb(string s) {
        int sum = 0, k = 0;
        for (int i = s.length() - 1; i >= 0; i--) {
            int digit = stoi(s.substr(i, 1));
            sum += digit * pow(2, k++);
        }
        return sum;
    }

    void solve() {
        int a = ni();
        int b = ni();
        cout << max(0, a - 2 * b) << endl;
    }
};

int main() {
    Main m;
    m.solve();
    return 0;
}
// End of Code
